src/topology/subset_properties.lean

```diff
@@ -917,6 +925,8 @@ lemma subtype.connected_space {s : set α} (h : is_connected s) :
   (subtype.preconnected_space h.is_preconnected).is_preconnected_univ,
   to_nonempty := h.nonempty.to_subtype }
 
+/-- A set is preconnected if and only if
+for every cover by two disjoint open sets, it is contained in one of the two covering sets. -/
```

Comment:
You write `disjoint open sets`, but in the statement they are only disjoint on `s`.